src/tests/suite/daos_container.c

```diff
@@ -2463,22 +2440,22 @@ co_api_compat(void **state)
 	uuid_t			uuid1;
 	uuid_t			uuid2;
 	char			*label = "test_api_compat_label1";
+	char			str[37];
 	daos_handle_t		coh;
 	daos_cont_info_t	info;
 	int			rc;
 
 	if (arg->myrank != 0)
 		return;
 
-	uuid_generate(uuid1);
```

Comment:
same as the DFS one, this is about checking API/ABI compatibility and think we should not make any changes to this particular test (co_api_compat)